components/plurals/tests/fixtures/operands.json

```diff
@@ -2,119 +2,127 @@
   "string": [
     {
       "input": "0",
-      "output": [0, 0, 0, 0, 0, 0]
+      "output": [0, 0, 0, 0, 0, 0, 0]
     },
     {
       "input": "2",
-      "output": [2, 2, 0, 0, 0, 0]
+      "output": [2, 2, 0, 0, 0, 0, 0]
     },
     {
       "input": "57",
-      "output": [57, 57, 0, 0, 0, 0]
+      "output": [57, 57, 0, 0, 0, 0, 0]
     },
     {
       "input": "987",
-      "output": [987, 987, 0, 0, 0, 0]
+      "output": [987, 987, 0, 0, 0, 0, 0]
     },
     {
       "input": "1234567",
-      "output": [1234567, 1234567, 0, 0, 0, 0]
+      "output": [1234567, 1234567, 0, 0, 0, 0, 0]
     },
     {
       "input": "-1000000",
-      "output": [1000000, 1000000, 0, 0, 0, 0]
+      "output": [1000000, 1000000, 0, 0, 0, 0, 0]
     },
     {
       "input": "0.23",
-      "output": [0.23, 0, 2, 2, 23, 23]
+      "output": [0.23, 0, 2, 2, 23, 23, 0]
     },
     {
       "input": "0.230",
-      "output": [0.23, 0, 3, 2, 230, 23]
+      "output": [0.23, 0, 3, 2, 230, 23, 0]
     },
     {
       "input": "23.00",
-      "output": [23, 23, 2, 0, 0, 0]
+      "output": [23, 23, 2, 0, 0, 0, 0]
     },
     {
       "input": "0.0203000",
-      "output": [0.0203, 0, 7, 4, 203000, 203]
+      "output": [0.0203, 0, 7, 4, 203000, 203, 0]
     },
     {
       "input": "123.45",
-      "output": [123.45, 123, 2, 2, 45, 45]
+      "output": [123.45, 123, 2, 2, 45, 45, 0]
     },
     {
       "input": "-1234.567",
-      "output": [1234.567, 1234, 3, 3, 567, 567]
+      "output": [1234.567, 1234, 3, 3, 567, 567, 0]
     },
     {
       "input": "2.813829837982735",
-      "output": [2.813829837982735, 2, 15, 15, 813829837982735, 813829837982735]
+      "output": [2.813829837982735, 2, 15, 15, 813829837982735, 813829837982735, 0]
     },
     {
       "input": "2.0",
-      "output": [2, 2, 1, 0, 0, 0]
+      "output": [2, 2, 1, 0, 0, 0, 0]
+    },
+    {
+      "input": "2e3",
+      "output": [2, 2, 0, 0, 0, 0, 3]
+    },
+    {
+      "input": "2.0e12",
```

Comment:
Worth adding a test case for negative exponents? I think they're unlikely, but we should know how we behave (return an error? set to zero?)

Reply:
I considered where to put it, but since UTS doesn't allow for them, I would only do this by extending our data driven `FromStr` tests, and we're planning to move that to `FixedDecimal` so I decided to not spend time on that. Instead, I hope we'll have such tests in `FromStr for FixedDecimal`.

Let me know if you believe we should invest time in rebuilding our `FromStr` test corpus or operands in Plurals.